packages/gamut/src/GridForm/GridFormInputGroup/index.tsx

```diff
@@ -22,24 +22,59 @@ export type GridFormInputGroupProps = {
   setValue: (value: any) => void;
 };
 
+const StyledFormGroup = styled(FormGroup)`
+  margin-bottom: 0;
+`;
+
+const StyledGridFormCheckboxInput = styled(GridFormCheckboxInput)`
+  width: 100%;
+`;
+
+const StyledGridFormCustomInput = styled(GridFormCustomInput)`
+  width: 100%;
+`;
+
+const StyledGridFormFileInput = styled(GridFormFileInput)`
+  width: 100%;
+`;
+
+const StyledGridFormRadioGroupInput = styled(GridFormRadioGroupInput)`
+  width: 100%;
+`;
+
+const StyledGridFormSelectInput = styled(GridFormSelectInput)`
+  width: 100%;
+`;
+
+const StyledGridFormTextArea = styled(GridFormTextArea)`
+  width: 100%;
+`;
+
+const StyledGridFormTextInput = styled(GridFormTextInput)`
+  width: 100%;
+`;
+
+const StyledFormGroupLabel = styled(FormGroupLabel)`
+  display: inline-block;
+  margin-right: 0.5rem;
+`;
```

Comment:
```suggestion
const StyledFormGroup = styled(FormGroup)`
  margin-bottom: 0;
  
  // This is always the input
  > *:last-child {
   width: 100%;
  }
  
  ${FormGroupLabel} {
    display: inline-block;
    margin-right: 0.5rem;
   }
`;
```
Since these components are not used outside of this content it may make sense for us to let this wrapper instruct its direct descendant to be 100%.

Reply:
Haha I was actually going to suggest a different approach:

```suggestion
const StyledFormGroup = styled(FormGroup)`
  margin-bottom: 0;
`;

const inputStyles = `width: 100%`;

const StyledGridFormCheckboxInput = styled(GridFormCheckboxInput)(inputStyles)(inputStyles);
const StyledGridFormCustomInput = styled(GridFormCustomInput)(inputStyles);
const StyledGridFormFileInput = styled(GridFormFileInput)(inputStyles);
const StyledGridFormRadioGroupInput = styled(GridFormRadioGroupInput)(inputStyles);
const StyledGridFormSelectInput = styled(GridFormSelectInput)(inputStyles);
const StyledGridFormTextArea = styled(GridFormTextArea)(inputStyles);
const StyledGridFormTextInput = styled(GridFormTextInput)(inputStyles);

const StyledFormGroupLabel = styled(FormGroupLabel)`
  display: inline-block;
  margin-right: 0.5rem;
`;
```

...no preference from me.

Reply:
ooh that looks clean but went with the fancy css selector instead this time around.